resources/js/components/GlobalSearch.vue

```diff
@@ -0,0 +1,282 @@
+<script>
+import ThemeWidget from './Theme/ThemeWidget';
+import PersonWidget from './PersonWidget';
+import MapWidget from './Map/MapWidget';
+import { ContentLoader } from 'vue-content-loader';
+import location from '../api/location';
+import people from '../api/people';
+import topics from '../api/topics';
+import intersectionObserverMixin from '../mixins/intersectionObserverMixin';
+export default {
+    name: 'GlobalSearch',
+    mixins: [
+        intersectionObserverMixin
+    ],
+    components: {
+        ThemeWidget,
+        PersonWidget,
+        MapWidget,
+        ContentLoader
+    },
+    data () {
+        return {
+            onFocus: false,
+            loading: false,
+            observableBlock: '.ris-load-element',
+            inputPlaceholder: 'Suche nach Themen, Vorlagen, Sitzungen...',
+            optionsList: [
+                'Bewohnerparken',
+                'Radwege',
+                'Brückensanierung',
+                'Umweltzone',
+                'Öffentliche Parkplätze',
+                'Schulen',
+                'Klimanotstand',
+                'eRoller'
+            ],
+            inputValue: '',
+            activeTab: 'Alle',
+            topicsList: [],
+            topicsCount: 0,
```

Comment:
i would recommend to rename this and other `...Count` properties, since at first i thought its just for example `this.topicsList.length` but in real its total count returned from backend, so i suggest to name it  like`...totalCount` or even leave a comment

Reply:
okay, Ill rename all these vars to ...TotalCount